Mathlib/RingTheory/Ideal/Span.lean

```diff
@@ -257,3 +257,36 @@ theorem ker_toSpanSingleton_one_sub_eq_span :
   rw [ker_toSpanSingleton_eq_span he.one_sub, sub_sub_cancel]
 
 end IsIdempotentElem
+
+namespace Subideal
+
+open Subtype
+
+variable {A : Type*} [CommSemiring A] {I : Ideal A}
+
+instance : CompleteLattice { J : Ideal A // J ≤ I } := Set.Iic.instCompleteLattice
```

Comment:
Rather than adding this instance, it might make sense to directly use `Set.Iic I` rather than `{ J : Ideal A // J ≤ I }` in your application. Then you shouldn't need any ideal-specific API